static/badges/Test-Canceled-lightgrey.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="92" height="20"><linearGradient id="b" x2="0" y2="100%"><stop offset="0" stop-color="#bbb" stop-opacity=".1"/><stop offset="1" stop-opacity=".1"/></linearGradient><clipPath id="a"><rect width="92" height="20" rx="3" fill="#fff"/></clipPath><g clip-path="url(#a)"><path fill="#555" d="M0 0h33v20H0z"/><path fill="#9f9f9f" d="M33 0h59v20H33z"/><path fill="url(#b)" d="M0 0h92v20H0z"/></g><g fill="#fff" text-anchor="middle" font-family="DejaVu Sans,Verdana,Geneva,sans-serif" font-size="110"><text x="175" y="150" fill="#010101" fill-opacity=".3" transform="scale(.1)" textLength="230">Test</text><text x="175" y="140" transform="scale(.1)" textLength="230">Test</text><text x="615" y="150" fill="#010101" fill-opacity=".3" transform="scale(.1)" textLength="490">Canceled</text><text x="615" y="140" transform="scale(.1)" textLength="490">Canceled</text></g> </svg>
```

Comment:
Did you create these images yourself?

Reply:
I generated these images in shields.io and then i save them in my local (as a svg)

Reply:
You're lucky that shields.io doesn't claim any copyright on the generated badges.